### DISCUSSION TITLE
loc_frame support in v2.0.3

### DISCUSSION THREAD

**Discussion by zrzrv5** [deepmodeling/deepmd-kit#1244] [Q&A]
Hi,

I just updated deepmd-kit from v1.3.3 to v2.03 on my local offline workstation, and I can't train a model using `loc_frame` descriptor. When I train the model using `loc_frame`, the code in `entrypoints/train.py` checks jdata['sel'], but `loc_frame` only has `sel_a` and `sel_r`.
So I made the following changes on my machine:
  1. skip update_sel(jdata) in `entrypoints/train.py line 90`
  2. add @Descriptor.register('loc_frame') in `descriptor/loc_frame.py line8`

Now I can start the training, but lammps crashes when using this model. I also tried converting a model from 1.3.3, also doesn't work in lammps.

Is loc_frame not available in this version? Or do I need to change other codes?

Also, is there any way to check the lammps and deeppot error log?

Thanks,

**Comment by wanghan-iapcm** [comment]
@njzjz could you please help checking the bug?

**Reply by njzjz** [reply]
I'll take a look.
